tutorials/pso_example.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+import random
+import sys
+
+from mpi4py import MPI
+
+from propulate import Islands
+from propulate.propagators import Conditional, StatelessPSO
+from function_benchmark import get_function_search_space
+from propulate.propagators.pso import (
+    BasicPSO,
+    VelocityClamping,
+    Constriction,
+    Canonical,
+    PSOInitUniform,
+)
+
+############
+# SETTINGS #
+############
+
+function_name = sys.argv[
+    1
+]  # Get function to optimize from command-line. Possible Options: See function_benchmark.py
+NUM_GENERATIONS: int = int(sys.argv[2])  # Set number of generations.
```

Comment:
This should be done with an argparser. See, e.g., ``propulator_example.py``.

Reply:
Yes, I am with you. 
Currently, I don't have capacities for this.

So I would kindly ask you to keep this as a todo for in two weeks the earliest.

Reply:
Most of the argparsing stuff (at least the common) should be externalized into another file.